centreon/tests/e2e/features/Migration-scripts/02-media.feature

```diff
@@ -0,0 +1,69 @@
+@ignore
+@REQ_MON-24294
+
+Feature: Migration of medias from a source platform to a target platform
+
+  Background:
+    Given an admin user who wants to migrate medias from a source platform to a target platform
+    And the user has access to both the source and target platforms
+    And another non-admin user has the necessary rights to manage medias
+    And the non-admin user has access to both the source and target platforms
+
+  Scenario: Medias creation on the source platform
+    Given a non-admin user logged in on the source platform
+    When the user adds a new media
+    Then the media is added and listed on the source platform
```

Comment:
This scenario should be put under feature "media management" when the feature will be processed with gherkin.